packages/sync/src/class-defaults.php

```diff
@@ -1129,6 +1128,13 @@ public static function is_multi_network() {
 	 */
 	public static $default_max_queue_size_full_sync = 1000; // max number of total items in the full sync queue.
 
+	/**
+	 * Default max time for sending in immediate mode.
+	 *
+	 * @var float Number of Seconds
+	 */
+	public static $default_full_sync_send_duration = 9;
```

Comment:
Can we change this somehow via the sync setting endpoint?

Reply:
yes we can